README.md

```diff
@@ -1,4 +1,10 @@
 # iRODS-User-Training
 Slides, exercises and extra files for the iRODS course at KU Leuven
 
-More documentation about the KU Leuven HPC infrastructure can be found [here](https://docs.vscentrum.be/en/data_m/). In the bottom corner, switch 'latest' to 'data_m' to view more documentation on the Tier-1 data service.
+This training explains the basics of the VSC Tier-1 Data Service.
+
+More documentation about the KU Leuven HPC infrastructure can be found [here](https://docs.vscentrum.be/en/data_m/).  
+In the bottom corner, make sure the branch is set to 'data_m' instead of 'latest' to view more documentation on the Tier-1 data service.
```

Comment:
Shall we remove the "make sure the branch is set to 'data_m'"? Because the link doesn't show it anymore, right!
Thanks.

Reply:
Indeed, see next commit.